misc/dist/html/full-size.html

```diff
@@ -133,7 +133,9 @@
 		</div>
 		<div id='status-notice' class='godot' style='display: none;'></div>
 	</div>
-
+	<script>
+		const CACHE_BUSTER="$CACHE_BUSTER";
```

Comment:
Can this be moved to the inline script entry below (the one that starts with `const GODOT_CONFIG`)?

Reply:
This wasn't resolved fully.

Reply:
Oh I forgot to comment. it is very tricky to move the line to the godot_config. The script in between needs it (the line src='$GODOT_URL'>). You can better not declare something after it is used in javascript. This would also make it a bit unclear what is happening.